packages/core/cli/src/__template__/sandbox/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "baseUrl": "."
+    "baseUrl": "."{%%SANDBOX_TSCONFIG_PATHS%%
+    }
```

Comment:
```suggestion
    "baseUrl": "."{%%SANDBOX_TSCONFIG_PATHS%%}
```

The placeholder is broken due to the whitespaces.

Reply:
This is just the template file; formatting will be wrong here it's just used to generate the actual file

Reply:
Good catch, thanks!